recipes/ghcup/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ghcup" %}
+{% set version = "0.1.17.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.haskell.org/~ghcup/{{ version }}/aarch64-apple-darwin-ghcup-{{ version }}   # [osx and arm64]
+  sha256: e25730a1b6a95bf4548f92cd84906955474a9243445a636b8d63e5a89a557db2    # [osx and arm64]
+  url: https://downloads.haskell.org/~ghcup/{{ version }}/aarch64-linux-ghcup-{{ version }}   # [linux and aarch64]
+  sha256: 35899ab2338b52fac84d49f16edbbb7701498e93ee8012309c7d6f2f138bb993   # [linux and aarch64]
+  url: https://downloads.haskell.org/~ghcup/0.1.17.5/x86_64-apple-darwin-ghcup-0.1.17.5    # [osx and x86_64]
+  sha256: 8b5159775de08a5f268f4be217fbb6f427ff8d8977197af209211b81ca0fe27f  # [osx and x86_64]
+  url: https://downloads.haskell.org/~ghcup/0.1.17.5/x86_64-linux-ghcup-0.1.17.5  # [linux and x86_64]
+  sha256: 7541bcf0b402f99d05cd1937f9fef69aec45c3153e42606671825be248484fb5  # [linux and x86_64]
+  url: https://downloads.haskell.org/~ghcup/0.1.17.5/x86_64-mingw64-ghcup-0.1.17.5.exe  # [win]
+  sha256: 7a6f0264c097044ede7d4208e153cf9fc6e06e49525e237dc32e2abd87ad3430   # [win]
+  fn: ghcup  # [not win]
+  fn: ghcup.exe   # [win]
+
+build:
+  number: 0
+
+test:
+  commands:
+    - ghcup -h
+
+about:
+  home: https://gitlab.haskell.org/haskell/ghcup-hs
+  summary: An installer for the general purpose language Haskell
+  license: LGPL-3.0-only
```

Comment:
Is this enough? What about the dependencies?

Reply:
I'm not sure how to even go about finding these. What do you recommend?

Reply:
Ask upstream about it? They might know

Reply:
Looking at the repo I could not find any reference to any other license.

Reply:
Please don't merge with unresolved conversations. This package vendors in other haskell packages just like rust/go and we have mechanisms for those packages.